google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/CloudApiManagementConfirmationDialog.java

```diff
@@ -68,6 +117,14 @@
 
     populateLibraryList(apisToEnableList, apisToEnable);
     populateLibraryList(apisNotSelectedToEnableList, apisNotToEnable);
+
+    newServiceAccountCheckbox.addActionListener(
+        e -> serviceAccountDetailsPane.setVisible(((JCheckBox) e.getSource()).isSelected()));
```

Comment:
Nit: this event handler feels a bit hidden in the setup code, maybe we could call a method, something like `handleNewServiceAccountClick()`? Might make it a bit easier to find what happens.